config/test_dtm_config.yaml

```diff
@@ -24,3 +24,7 @@ test_41230:
 
 test_42246:
   num_loop: 300
+
+test_42247:
+  nclients: 10
```

Comment:
clients and samples are declared at start of this file. 
That can be used for all the tests.

Reply:
1 client and 500 samples will be too much for IOs during continuous rgw process restarts.
I am using this config for cont reads and writes.